mm2src/coins/nft/storage/wasm/wasm_storage.rs

```diff
@@ -959,9 +956,9 @@ pub(crate) struct NftTransferHistoryTable {
 }
 
 impl NftTransferHistoryTable {
-    // old prim key index for DB_VERSION: u32 = 1
-    const CHAIN_TX_HASH_LOG_INDEX_INDEX: &'static str = "chain_tx_hash_log_index_index";
-    // this is prim key multi index for DB_VERSION = 2
+    // old prim key index for DB_VERSION = 1
+    const _CHAIN_TX_HASH_LOG_INDEX_INDEX: &'static str = "chain_tx_hash_log_index_index";
```

Comment:
do we need to keep this?

Reply:
It is used, please switch to up to date changes
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2209/files#diff-c2d30614cd223699835e416f1071343c867300a53802fee1ab41517910df505aR959-R960